src/commands/selectCommandTemplate.ts

```diff
@@ -166,16 +165,7 @@ export async function selectCommandTemplate(
     actionContext.telemetry.properties.commandContextType = `[${selectedTemplate.contextTypes?.join(', ') ?? ''}]`;
     actionContext.telemetry.properties.currentContextType = currentContextType;
 
-    let resolvedCommand = resolveVariables(selectedTemplate.template, folder, additionalVariables);

```

Comment:
Since the command template is now based on the setting, this is no longer needed to rewrite commands to match what they used in settings.